src/overlays/actors/ovl_Dm_Ah/z_dm_ah.c

```diff
@@ -108,25 +108,23 @@ s32 func_80C1D6E0(DmAh* this, PlayState* play) {
     return true;
 }
 
-Actor* func_80C1D78C(PlayState* play) {
-    Actor* tempActor;
-    Actor* foundActor = NULL;
+Actor* DmAh_FindAnjuActor(PlayState* play) {
+    Actor* actorIter = NULL;
 
     while (true) {
-        foundActor = SubS_FindActor(play, foundActor, ACTORCAT_NPC, ACTOR_DM_AN);
+        actorIter = SubS_FindActor(play, actorIter, ACTORCAT_NPC, ACTOR_DM_AN);
 
-        if ((foundActor == NULL) || (foundActor->update != NULL)) {
+        if ((actorIter == NULL) || (actorIter->update != NULL)) {
             break;
         }
```

Comment:
I can't speak for Engineer but my guess is that this didn't get split because it only has 1 conditional, while the ones that did get split have two conditionals with an `&&`. Tbh, I'd agree either split all of them (even these simple ones) or keep them all together. I'm not sure I really have a preference which.

Reply:
Yeah exactly I only focused on the ones with the 3 conditions, and just left the 2 conditions alone. But I went and split the 2-conditions one as well